packages/cozy-dataproxy-lib/src/api.ts

```diff
@@ -0,0 +1,2 @@
+export { SearchEngine } from './search/SearchEngine'
+export { DataProxyProvider, useDataProxy } from './dataproxy/DataProxyProvider'
```

Comment:
Are `DataProxyProvider` and `useDataProxy` necessary here? I think they are currently only used inside the ui search

Reply:
I can move them if they are more linked to the ui part.

Reply:
I'm not sure, as I guess the provider can be seen as an api, more than a ui concept. Let's keep it like this, but I think you can remove the `useDataProxy` export

Reply:
Agree, even it will be used by ui components, it's more an api concept 👍